client.go

```diff
@@ -37,7 +37,17 @@ func callAutograph(auth authorization, body []byte, xff string) (signedBody []by
 		KeyID: auth.Signer,
 	}
 	if auth.AddonID != "" {
-		request.Options = xpi.Options{ID: auth.AddonID}
+		opt := xpi.Options{
+			ID: auth.AddonID,
+			PKCS7Digest: "SHA1",
+		}
+		if auth.AddonPKCS7Digest != "" {
+			opt.PKCS7Digest = auth.AddonPKCS7Digest
```

Comment:
autograph handles this at https://github.com/mozilla-services/autograph/blob/2.2.0/signer/xpi/xpi.go#L329-L343

I'd rather leave it as is to avoid having to update it in two places